HospitalFrontend/src/components/ExaminationItem.vue

```diff
@@ -0,0 +1,128 @@
+<template>
+    <div class="bg-light row rounded border border-info mt-1">
+        <table class="table col-9 h-100">
+            <thead>
+                <tr class="table-info">
+                    <th scope="col">
+                        Start time
+                    </th>
+                    <th scope="col">
+                        Room
+                    </th>
+                    <th scope="col">
+                        Doctor
+                    </th>
+                </tr>
+            </thead>
+            <tbody>
+                <tr>
+                    <td class="align-middle" rowspan="2">
+                        <span>{{getDate(examination.timeInterval.start)}}</span><br/>
+                        <span class="lead">{{getTime(examination.timeInterval.start)}}</span>
+                    </td>
+                    <td>
+                        {{examination.room.name}}
+                    </td>
+                    <td>
+                        {{examination.doctor.person.name + " " + examination.doctor.person.surname}}
+                    </td>
+                </tr>
+                <tr>
+                    <td>
+                        {{examination.room.department.description}}
+                    </td>
+                    <td>
+                        {{examination.doctor.department.description}}
+                    </td>
+                </tr>
+            </tbody>
+        </table>
+
+
+        <button class="btn btn-info col" v-on:click="cancelAppointment(examination.id)" v-if="isCancelable()">Cancel appointment</button>
+        <div class="col text-danger lead font-weight-bold d-flex justify-content-center" v-else-if="examination.isCanceled">
+            <h3 class="align-self-center">Canceled</h3>
+        </div>
+
+        <button v-else-if="!surveyCompleted" class="btn btn-success col" v-on:click="takeSurvey">Take a survey</button>
+
+        <div v-else class="col text-info lead font-weight-bold d-flex justify-content-center">
+            <h3 class="align-self-center">Survey completed</h3>
+        </div>
+
+    </div>
+</template>
+
+<script>
+import Toastify from 'toastify-js'
+import axios from 'axios'
+import api from '../constant/api.js'
+
+export default {
+    name: "ExaminationItem",
+    props: {
+        examination: Object,
+        surveyCompleted: Boolean
+    },
+    emits: ['updateExaminations'],
+    methods: {
+        getDate: function(date) {
+            console.log(this.surveyCompleted)
+            let dateObj = new Date(date)
```

Comment:
Logging repeats at 73rd line in ObservePatientExaminations.vue.

Reply:
Logs removed.